site/source/tutorials/005_form_widgets/index.md

```diff
@@ -29,7 +29,33 @@ The first step to allowing the user to create new workers is to create a form. T
 
 {% instruction 'Add the following to `WorkerForm.ts`.' %}
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:1-4,7,14-15,19-27,29,42-43,48-51,85-87 %}
+```typescript
```

Comment:
Any reason to not still have this reside in a separate file?

Reply:
As far as I understand it, we only use the initial or finished source files as code files. Unfortunately one of the import lines from the final file doesn't compile at this stage so I can't cut a combination of lines that will work. I would have to split the `v` and `w` imports into separate lines in the final file.